spec.html

```diff
@@ -27915,14 +27916,12 @@ <h1>Math.log2 ( _x_ )</h1>
       </emu-clause>
 
       <emu-clause id="sec-math.max">
-        <h1>Math.max ( _value1_, _value2_, ..._values_ )</h1>
-        <p>Given zero or more arguments, calls ToNumber on each of the arguments and returns the largest of the resulting values.</p>
```

Comment:
Is there a reason to remove this sentence?

Reply:
Nah, I'll put it back.